README.md

```diff
@@ -4,9 +4,10 @@ ament_nodl
 Ament extension for exporting NoDL .xml files
 
 Usage
+
 ---
 With a file named `nodl.xml` in the same folder as your project's `CMakeLists.txt`, add the following line to your `CMakeLists.txt` before calling `ament_package()`
 
 ```CMake
 nodl_export_node_description_file(nodl.xml)
-```
\ No newline at end of file
+```
```

Comment:
Does this file need to be part of this diff?

Reply:
It'd be nice to have the markdowns cleaned up for release, it messies up this commit but its too small to warrant a PR of its own

Reply:
Alright.